src/tox_ini_fmt/__main__.py

```diff
@@ -23,6 +37,35 @@ def color_diff(diff: Iterable[str]) -> Iterable[str]:
             yield line
 
 
+# partly copied from: https://github.com/pre-commit/pre-commit-hooks/blob/main/pre_commit_hooks/mixed_line_ending.py
+def detect_line_ending(filename: str) -> str:
+    with open(filename, "rb") as f:
```

Comment:
single charachter variables should be only used by comprehensions, f -> file_handler; but why are we not using pathlib read_bytes to read this instead?